applications/nydus/nydusdfile/serverstart.sh

```diff
@@ -3,6 +3,36 @@
 #nydusdir need be scp
 rm -rf $2
 mkdir $2
+nydusdconfig='
+{\n
+  "device": {\n
+    "backend": {\n
+      "type": "registry",\n
```

Comment:
here i think we can set  fixed "127.0.0.1" in  httpserver.json. and use `sed -i "s/127.0.0.1/$3/g" ./nydusd_scp_file/httpserver.json` to modify lcoal ip.